docs/concepts/cache.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 id: cache
-title: Low-Latency and Eventual Consistency
+title: Low-Latency Edge Network
+sidebar_label: Low-Latency
 ---
 
-Some Ory Network services provide low-latency APIs. These APIs are eventually consistent, meaning that changes you make might not
-be immediately reflected in the API.
+The Ory Network provides a global edge-network to provide low-latency access to critical-path APIs. These APIs are eventually
+consistent, meaning that changes you make might not be immediately reflected in the API.
```

Comment:
```suggestion
The Ory Network provides a global Edge Network to ensure low-latency access to critical-path APIs. These APIs are eventually
consistent, which means that the changes you make are always reflected in the API, even if in some cases this might not be instantaneous.
```

No hyphen in "Edge Network". Capitalized since it seems to be used as a proper term. Also, Google does that:
https://peering.google.com/#/

Reply:
Applied in another changeset

Reply:
Sorry, but I don't think this was applied. Something got funked in the process? 🤔